common/goos/PrettyPrinter.cpp

```diff
@@ -33,7 +33,7 @@ goos::Object float_representation(float value) {
   u8 exp = (int_value >> 23) & 0xff;
   u32 mant = int_value & 0x7fffff;
   if ((exp == 0 && mant != 0) || exp == 0xff) {
-    lg::warn("PS2-incompatible float (0x{:08X}) detected! Writing as the-as cast.", int_value);
+    // lg::warn("PS2-incompatible float (0x{:08X}) detected! Writing as the-as cast.", int_value);
```

Comment:
should we just remove this log all together?  I also had it commented out...is this some sort of hack that we are weary of -- because it seems to happen alot

Reply:
As far as I can tell it only happens in the particle definitions which are kind of a disaster right now